pkg/config/mesh/watcher.go

```diff
@@ -94,6 +98,40 @@ func (w *InternalWatcher) AddMeshHandler(h func()) {
 	w.handlers = append(w.handlers, h)
 }
 
+func (w *InternalWatcher) HandleMeshConfigData(yaml string) {
+	w.revMeshConfig = yaml
+	w.HandleMeshConfig(w.merged())
+}
+
+func (w *InternalWatcher) HandleUserMeshConfig(yaml string) {
+	w.userMeshConfig = yaml
+	if w.revMeshConfig != "" {
```

Comment:
Why do we have this check? As far as I can tell, this means that if there is no mesh configmap (which is possible, although not in the default install), changes to user mesh will not trigger an update

Reply:
Yes, we continue to require the 'per revision' configmap, created during install. 
It is valid to have only revision. If per-revision config map missing - we continue the current behavior ( use Defaults )